reading/livebook.livemd

```diff
@@ -203,9 +203,9 @@ Sometimes order of evaluation can cause issues in Exercises by using a stale ver
 
 ## Formatting
 
-You can format Elixir code using the <kbd>ALT</kbd>+<kbd>SHIFT</kbd>+<kbd>F</kbd> (Windows & Linux) Keybinding or <kbd>SHIFT</kbd>+<kbd>OPTION</kbd>+<kbd>F</kbd> (MacOS).
+You can format Elixir code using the <kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>I</kbd> (Windows & Linux) Keybinding or <kbd>SHIFT</kbd>+<kbd>OPTION</kbd>+<kbd>I</kbd> (MacOS).
```

Comment:
I've noticed for most people, it's this command. However, for me it's alt-shift-f. 

Very odd. This looks like a good change, It might be word directing folks to the Keybinding section in Livebook 🤔